data_architect_misc/data_transformer/transform_functions/common_comp_harm_transform_functions.py

```diff
@@ -454,7 +480,7 @@ def add_HARMONIZED_GROSS_SPEND_column(
             from the original (raw) gross spend column trimmed to just
             two decimal digits.
         """
-        df[comp_harm_constants.GROSS_SPEND_COLUMN] = df[existing_gross_spend_col_name]
+        df[comp_harm_constants.GROSS_SPEND_COLUMN] = df[existing_gross_spend_col_name].astype(float)
```

Comment:
Is that because the existing gross spend data from raw files are read as string in Nicaragua, El Salvador, etc.?